src/runtime/policy/ingress-validation.ts

```diff
@@ -248,41 +248,46 @@ export class IngressValidation {
   //   Blah -> null
   //   (Foo, Blah) -> null
   //   ...
-  public getMaxReadType(type: Type): Type|null {
+  public getMaxReadType(type: Type, errors?: string[]): Type|null {
```

Comment:
there is an `IngressValidationResult` class declared later in the file, maybe it can be used for `errors` here as well?

Reply:
I did consider reusing `IngressValidationResult`, but decided to keep it simple for now.